src/services/administration/store/policy/type.ts

```diff
@@ -1,7 +1,15 @@
 import { PolicyDataModel } from '@/services/administration/iam/policy/lib/type';
 
+export interface PolicyListState {
+	results: PolicyDataModel[];
+	total_count: number;
+}
+
 export interface PolicyState {
 	policyData: Partial<PolicyDataModel>|null;
+	policyListLoading: boolean;
+	policyList?: PolicyDataModel[];
+	total_count?: number;
```

Comment:
```suggestion
	totalCount: number;
```

1. console에서 관리하는 모든 데이터는 camel case 입니다!
2. undefined를 허용하지 않고 항상 0으로 넣어주어도 될 것 같습니다.